tests/benchmarks/benchmarks.py

```diff
@@ -0,0 +1,111 @@
+import json
```

Comment:
Should we move this file to the `integration/` folder since this isn't run with the unit tests?